### DISCUSSION TITLE
How to open kitty right in the middle on my screen?

### DISCUSSION THREAD

**Discussion by JackYansongLi** [hyprwm/Hyprland#3518] [Q&A]
I want to open kitty right in the middle on my screen, so I read the document and tried for a long time. I have tried several settings, such as: 
```console
windowrule = size 300x300, center, title:^(kitty)$
bind = SUPER, RETURN, exec, [float]kitty --title kitty
```
and
```console
windowrule=float,title:^(kitty_floats)$
bind=SUPER,RETURN,exec,kitty --title kitty_floats
```
None of the settings did what I want. Anyone knows how to set or where I can read some settings that can satisfy my requirement?

**Comment by Aqa-Ib** [comment]
windowrulev2 = float, class:^(kitty)$
windowrulev2 = center, class:^(kitty)$

**Comment by MightyPlaza** [comment]
`bind = SUPER, RETURN, exec, [float; center] kitty`

**Reply by Aqa-Ib** [reply]
nice one, where is that documented ?

**Reply by MightyPlaza** [reply]
https://wiki.hyprland.org/Configuring/Dispatchers/#executing-with-rules

**Reply by Aqa-Ib** [reply]
Thank you!
